Scripts/Python/psnlYeeshaPageVisController.py

```diff
@@ -0,0 +1,141 @@
+NVIDIA PhysX SDK, Microsoft DirectX SDK, OpenSSL library, Independent
+JPEG Group JPEG library, Microsoft Windows Media SDK, or Apple QuickTime SDK
+(or a modified version of those libraries),
+containing parts covered by the terms of the Bink SDK EULA, 3ds Max EULA,
+PhysX SDK EULA, DirectX SDK EULA, OpenSSL and SSLeay licenses, IJG
+JPEG Library README, Windows Media SDK EULA, or QuickTime SDK EULA, the
+licensors of this Program grant you additional
+permission to convey the resulting work. Corresponding Source for a
+non-source form of such a combination shall include the source code for
+the parts of OpenSSL and IJG JPEG Library used as well as that of the covered
+work.
+
+You can contact Cyan Worlds, Inc. by email legal@cyan.com
+ or by snail mail at:
+      Cyan Worlds, Inc.
+      14617 N Newport Hwy
+      Mead, WA   99021
+
+ *==LICENSE==* """
+
+from Plasma import *
+from PlasmaTypes import *
+from PlasmaVaultConstants import *
+from PlasmaNetConstants import *
+from xPsnlVaultSDL import *
+
+PageNumber = ptAttribInt(1, "Yeesha Page Number")
+stringShowStates = ptAttribString(2,"States in which shown")
+hideObjectsTrue = ptAttribSceneobjectList(3, "Hide Objects in List when States are true")
+hideObjectsFalse = ptAttribSceneobjectList(4, "Hide Objects in List when States are false")
+
+
+class psnlYeeshaPageVisController(ptMultiModifier):
+    def __init__(self):
+        ptMultiModifier.__init__(self)
+        self.id = 53000
+        version = 0
+        self.version = version
+        PtDebugPrint("__init__psnlYeeshaPageVisController v%d.%d" % (version,1),level=kWarningLevel)
+
+# Yeesha Pages available:
+#
+#0 (YeeshaPage01) sun and moon addition 
+#1 (YeeshaPage02) waterfall addition 
+#2 (YeeshaPage03) hut decal / interior rug addition 
+#3 (YeeshaPage04a) hut roof modification (swap) 
+#4 (YeeshaPage05) jumping pinnacles addition (swap) 
+#5 (YeeshaPage06) man-made dock addition 
+#6 (YeeshaPage07) kickable physical addition 
+#7 (YeeshaPage08) imager addition (needs KI wiring) 
+#8 (YeeshaPage09) music player 
+#9 (YeeshaPage10a - j) treegate tree (multi-swap)
+#11 (YeeshaPage12) weather stuff
+#~ psnlZandiVis (all Ypages, plus chair, shirt, books...) 
+#~ (YeeshaPage20) page the second bookcase (swap)
+#~ (YeeshaPage13) butterflies
+#~ (YeeshaPage14) fireplace
+#~ (YeeshaPage15) bench
+#~ (YeeshaPage16) firemarbles
+#~ (YeeshaPage17) lush
+#~ (YeeshaPage18) clock
+#~ (YeeshaPage19) birds
+#~ (YeeshaPage20) bridge to calendar pinnacle
+#~ (YeeshaPage21) leaf (maple trees)
+#~ (YeeshaPage22) grass
+#~ (YeeshaPage24) thunderstorm
+#~ (YeeshaPage25) Bahro poles/totems
+#~ (YeeshaPage26) Veelay Tsahvahn night skybox
+
+#Meaning of SDL values for each Yeesha Page:
+#
+#~ 0 - Page not found
+#~ 1 - Page found and active
+#~ 2 - Page found and inactive
+#~ 3 - Page found, active, and pending inactive when age emptied
+#~ 4 - Page found, inactive, and pending active when age emptied
```

Comment:
Can we move this outside of the class definition? As it stands, this breaks the indentation rules and may suddenly become an error in the future.